virttest/libvirt_vm.py

```diff
@@ -3223,6 +3223,7 @@ def _is_ga_finished():
 
         def _start_ga():
             if not _is_ga_running():
+                self.install_package("qemu-guest-agent")
```

Comment:
Hi, @yanqzhan I think _start_ga() function is just for start guest agent service. There are other function for install qemu-guest-agent function. It is very clear in the every function. If some cases failed because of the package is not installed. I think we should do some changes in the cases, e.g., to make sure the qemu-guest-agent pkgs are installed in the guest.

Reply:
Yes, agree about "Every function plays well in its role". Thank you. But vm.prepare_guest_agent() is a powerfull function can do many things. I wouldn't like to use it just for one line installation. I'll make change in virsh_domifaddr.py later to simply add a installation in it.
Close this one.